app/modules/asset_groups/__init__.py

```diff
@@ -16,7 +16,13 @@ def init_app(app, **kwargs):
     api_v1.add_oauth_scope(
         'asset_groups:write', 'Provide write access to Asset_groups details'
     )
-
+    api_v1.add_oauth_scope(
```

Comment:
undecided if this is necessary/correct or if asset_groups auths should be used. Any opinion?